app/src/main/java/com/cmput301f20t21/bookfriends/entities/Request.java

```diff
@@ -0,0 +1,33 @@
+package com.cmput301f20t21.bookfriends.entities;
+
+import com.cmput301f20t21.bookfriends.enums.REQUEST_STATUS;
+
+public class Request {
+    private String id;
+    private String requesterId;
+    private String bookId;
+    private REQUEST_STATUS status;
```

Comment:
do we want to have the owner user id? I know this can be found in the book entity, but it would be better if we have that attribute flatten out here, which is also needed/faster when we query for "incoming requests".

Reply:
addressed in meeting.